extensions/webirc.md

```diff
@@ -21,20 +21,30 @@ When a user connects through an indirect connection to the IRC server, the user'
 ## Format
 The `WEBIRC` command MUST be the first command sent from the WebIRC gateway to the IRC server and MUST be sent before capability negotiation.
 
-The `WEBIRC` command takes four parameters: `password `, `gateway`, `hostname`, and `ip`.
+The `WEBIRC` command may take five parameters: `password `, `gateway`, `hostname`, `ip`, and `options`.
```

Comment:
"takes between four and five parameters" sounds more specific to me.

Reply:
yeah this was the only thing I was iffy on, will fix it